apps/wizard/metagpt.py

```diff
@@ -134,12 +190,25 @@ def get_actual_path(path: str) -> str:  # -> Any:
             file_content = f.read()
         metadata_new_updated = st_ace(file_content, **ACE_DEFAULT, key="modified")  # gpt_updater.metadata_new_str
     with col12:
-        # Form to export
-        with st.form("form_export"):
-            output_dir = st.text_input("Output path")
-            overwrite = st.toggle("Overwrite", value=False)
-            st.form_submit_button(
-                "Export new file",
-                on_click=export_metadata_file,
-                args=(metadata_new_updated, overwrite, output_dir, filepath),
-            )
+        with st.expander("Export metadata file", expanded=False):
+            # Form to export
+            with st.form("form_export"):
+                overwrite = st.toggle(
```

Comment:
this line isn't working as expected - still just append the gpt_ to the filename and saves it in the directory

Reply:
Copy-pasting from Veronika's comment:

> the --overwrite property isn't working as expected in wizard, and even after pressing overwrite I am pretty sure it still executes this line  output_file_path = os.path.join(output_dir, "gpt_" + os.path.basename(path_to_file)). I am not sure how this is linked in wizard so maybe you could take a look?

Reply:
I think this is now solved